src/HypothesisClient/Model/CanBeValidated.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace eLife\HypothesisClient\Model;
+
+use InvalidArgumentException;
+
+interface CanBeValidated
+{
+    /**
+     * @throws InvalidArgumentException
+     *
+     * @return bool
+     */
+    public function validate() : bool;
```

Comment:
This appears not to be used anywhere except inside the constructor of `User`, so could never be `false`? (Post-initialisation validation is a bad pattern anyway.)